### PR TITLE
[#2598] Fix: Fix HDFS IP resolution for Docker Desktop

### PR DESCRIPTION
**Pull request by zivali**
### What changes were proposed in this pull request?
Fix HDFS client IP address resolution when using Docker Desktop. Originally it may resolve to invalid domain name causing error.

### Why are the changes needed?
Fix: #2598

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Existing tests.

### CODE REVIEW

**Comment by zivali** [comment]
@diqiu50 Hi, could you help take a look when you have time? Thank you!

File: integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java [hunk 1]
```diff
@@ -111,6 +111,7 @@ public void startTrinoContainer(
             .withExtraHosts(
                 ImmutableMap.<String, String>builder()
                     .put("host.docker.internal", "host-gateway")
+                    .put("gravitino-ci-hive", hiveContainerIp)
```
Comment (diqiu50) [inline integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java:114]: I suggest using the constant HiveContainer.HOST_NAME in the HiveContainer class, and also in line 78.

Comment (zivali) [reply integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java:114]: @diqiu50 Thanks for the suggestion! Do you mean by add the following method to HiveContainer class and use `.put(hiveContainer.getHostName(), hiveContainerIp)`?
``` java
public String getHostName() {
  return HOST_NAME;
}
```

Comment (xunliu) [reply integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java:114]: hi @zivali Thank you for your fixed this issue.
I think you can directly use `.put(HiveContainer.HOST_NAME, hiveContainerIp)`

Comment (zivali) [reply integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java]: Thanks for the suggestion! Sorry I missed that.
I updated this PR. Could you help take a look again? Thank you.

**Review by diqiu50 — APPROVED** [review]
LGTM

**Pull request merged**